designer/fitting/dwipi.py

```diff
@@ -22,12 +22,17 @@
 # Suppress warnings because mitigations are applied in code
 warnings.filterwarnings("ignore")
 
+# Progress bar Properties
+tqdmWidth = 100  # Number of columns of progress bar
```

Comment:
```suggestion
tqdmWidth = 70  # Number of columns of progress bar
```

Reply:
Most terminal widths are 80, 100 is too wide.

Reply:
Had to make a couple other changes for 70. Done and merging.